terra/views.py

```diff
@@ -188,6 +198,13 @@ def get_queryset(self):
             return Unit.objects.filter(type="1")
         return Unit.objects.filter(manager=self.request.user.employee)
 
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        # For now get current fiscal year
+        # Override this by query params when we add historic data
+        context["current_fy"] = current_fiscal_year_int()
```

Comment:
Isn't this what we are trying to remove?

Reply:
Yes, I'll remove the comment.